models/recipe.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (sequelize, DataTypes) => {
+    const Recipe = sequelize.define("recipe", {
+        recipe_name: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            validate: { notEmpty: true, notNull: true }
+        },
+        ingrediants: {
+            type: DataTypes.TEXT,
+            allowNull: false,
+            validate: { notEmpty: true , notNull: true}
+        }, 
+        instructions: {
+            type: DataTypes.TEXT,
+            allowNull: false, 
+            validate: { notEmpty: true , notNull: true}
+        }
+    },
+        {
+            freezeTableName: true
```

Comment:
I see that you use this everywhere. Please make it a global rule as described [here](https://sequelize.org/docs/v6/core-concepts/model-basics/#:~:text=This%20behavior%20can%20also%20be%20defined%20globally%20for%20the%20sequelize%20instance%2C%20when%20it%20is%20created)